src/AssemblySharedInfoGenerator/AssemblySharedInfo.tt

```diff
@@ -69,6 +69,6 @@ using System.Runtime.InteropServices;
 <#+
 int MajorVersion = 2;
 int MinorVersion = 0;
-int BuildNumber = 0;
+int BuildNumber = 1;
 int RevisionNumber = ((int)(DateTime.UtcNow - new DateTime(2017,1,1)).TotalDays)*10+((int)DateTime.UtcNow.Hour)/3;
```

Comment:
should it say 2018?

Reply:
@ramramps Not now, it will scrw us on the number and all the existing 2.0 beta packages on nuget. Will not change this date until next major release I believe

Reply:
ok

Reply:
@QilongTang  can you put that information in the code as a comment or TODO.